packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -141,21 +162,12 @@ module.exports = {
         test: /\.json$/,
         loader: 'json'
       },
-      // "file" loader makes sure those assets get served by WebpackDevServer.
-      // When you `import` an asset, you get its (virtual) filename.
-      // In production, they would get copied to the `build` folder.
-      {
-        test: /\.(ico|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2)(\?.*)?$/,
-        loader: 'file',
-        query: {
-          name: 'static/media/[name].[hash:8].[ext]'
-        }
-      },
-      // "url" loader works just like "file" loader but it also embeds
-      // assets smaller than specified size as data URLs to avoid requests.
+      // Use url loader to load resources needed from outside the appSrc
+      // directory.
       {
-        test: /\.(mp4|webm|wav|mp3|m4a|aac|oga)(\?.*)?$/,
+        test: /\.(eot|otf|svg|ttf|woff|woff2)(\?.*)?$/,
```

Comment:
I think we'll be able to get rid of this separate loader for web fonts, since these file extensions are handled by the default `url-loader` above.

The only difference left seems to be that this config loads files outside the `src` folder. I'd say let's simplify the rules for handling files further:
* JavaScript is compiled with Babel only if it's in the `src` folder.
* All the other file types can be in any folder.

Then we can simply drop the special handling for web fonts, and the ways different file types are handled is easy to predict and understand.